ui-manchette/src/components/helpers.ts

```diff
@@ -1,10 +1,19 @@
-import type { OperationalPoint } from '@osrd-project/ui-spacetimechart/dist/lib/types';
+import type {
+  OperationalPoint,
+  SpaceTimeChartProps,
+} from '@osrd-project/ui-spacetimechart/dist/lib/types';
 
-import { BASE_KM_HEIGHT, BASE_OP_HEIGHT } from './consts';
-import type { OperationalPointType, StyledOperationalPointType } from '../types';
+import { BASE_KM_HEIGHT, BASE_OP_HEIGHT, MAX_TIME_WINDOW, MAX_ZOOM_X, MIN_ZOOM_X } from './consts';
+import type {
+  OperationalPointType,
+  ProjectPathTrainResult,
+  StyledOperationalPointType,
+} from '../types';
 
 export const positionMmToKm = (position: number) => Math.round((position / 1000000) * 10) / 10;
 
+export const msToS = (time: number) => time / 1000;
```

Comment:
shouldn't it be in a `utils.ts` ? (same for `positionMmToKm`)

Reply:
Yes you're right, I move them there :+1: